bin/all_sky_search/pycbc_bin_trigger_rates_dq

```diff
@@ -82,27 +88,25 @@ dq_logl = np.array([])
 for filename in args.dq_file:
     logging.info('Reading DQ file %s...', filename)
     dq_data = load_timeseries(filename, group=args.dq_channel)
-    dq_logl = np.concatenate((dq_logl,dq_data[:]))
-    dq_times = np.concatenate((dq_times,dq_data.sample_times))
+    dq_logl = np.concatenate((dq_logl, dq_data[:]))
+    dq_times = np.concatenate((dq_times, dq_data.sample_times))
     del dq_data
 
 # todo: make this configurable
 percent_bin = 0.5
 n_bins = int(100./percent_bin)
-percentiles = np.linspace(0,100,n_bins+1)
+percentiles = np.linspace(0, 100, n_bins+1)
 bin_times = np.zeros(n_bins)
-dq_percentiles = np.percentile(dq_logl,percentiles)[1:]
-
+dq_percentiles = np.percentile(dq_logl, percentiles)[1:]
 
 # seconds bin tells what bin each second ends up
-seconds_bin = np.array([n_bins - \
-                        len(dq_percentiles[dq_percentiles >= dq_ll]) \
+seconds_bin = np.array([n_bins - np.sum(dq_percentiles >= dq_ll)
                         for dq_ll in dq_logl]).astype('int')
 del dq_percentiles
 
 # bin times tells how much time ends up in each bin
-bin_times = np.array([len(seconds_bin[seconds_bin==(i)]) \
-                     for i in range(n_bins)]).astype('float')
+bin_times = np.array([np.sum(seconds_bin == i)
```

Comment:
is this effectively the same as the previous code which used `(i)` ?

Reply:
Yes it is the same, this was something my IDE was complaining about for code style

Reply:
I have no idea why `(i)` was there in the first place, except that I was a more inexperienced years ago when I first wrote this

Reply:
yes, well as someone with many more years of inexperience behind me this is not unknown ..